docs/collator-instructions.md

```diff
@@ -0,0 +1,177 @@
+# How to set up an InvArch Collator
+
+Make sure you are logged in as a user with root privileges, otherwise:
+
+``sudo su -``
+
+Create a new service user to run your collator service:
+`sudo useradd --no-create-home --shell /usr/sbin/nologin tinkernet-collator`
+
+Install some required stuff:
+
+``curl https://sh.rustup.rs -sSf | sh``
+(choose option 1 - Proceed with installation (default))
+You may need to restart your system before the next steps.
+```
+rustup install nightly-2021-11-02
+rustup target add wasm32-unknown-unknown --toolchain nightly-2021-11-02
+apt install cmake git clang libclang-dev
+```
+Type Y to proceed.
+
+Clone the repo:
+
+ ``git clone git@github.com:InvArch/InvArch-Node.git``
+
+Type the following command, and make sure you can see "gm-chain", if not then you likely did something wrong.
+
+  ``ls -la``
+
+Enter the repo and check out the most recent tagged release of code (https://github.com/GMorDIE/gm-chain/releases)
+
+```
+cd InvArch-Node
+git checkout v1.2.0
+```
+
+You need to compile the code, this will take quite a while depending on your system (30+ minutes is normal):
+ ``cargo build --release``
+
+Mid-way through compiling, you likely need to enter this command when prompted:
+
+``rustup target add wasm32-unknown-unknown``
+
+Resume compiling:
+
+``cargo build --release``
+
+Move the node executable to `usr/local/bin`, make it executable, and change ownership to our `tinkernet-collator` service user:
+```
+sudo mv ~/InvArch-Node/target/release/invarch-collator /usr/local/bin/tinkernet-collator
+sudo chmod +x /usr/local/bin/tinkernet-collator
+sudo chown tinkernet-collator:tinkernet-collator /usr/local/bin/tinkernet-collator
+```
+
+Create the base-path folder, copy the gm_chain "chainspec" into it, and give it the necessary permissions & ownership:
+```
+sudo mkdir /var/lib/tinkernet
+sudo cp ~/InvArch-Node/res/kusama/tinker-raw.json /var/lib/tinkernet/tinker-raw.json
+sudo chown -R tinkernet-collator:tinkernet-collator /var/lib/tinkernet
+```
+
+Create a systemd service file to run your collator (and automatically restart it):
+
+`sudo nano /etc/systemd/system/tinkernet-collator.service`
+
+Within that file, paste in the following:
+```
+[Unit]
+Description=Tinkernet (InvArch) Collator
+After=network-online.target
+Wants=network-online.target
+
+[Service]
+Type=simple
+User=tinkernet-collator
+Group=tinkernet-collator
+ExecStart=/usr/local/bin/tinkernet-collator \
+  --base-path /var/lib/tinkernet \
+  --collator \
+  --force-authoring \
+  --name "YOUR-COLLATOR-NAME-HERE" \
+  --chain /var/lib/tinkernet/tinker-raw.json \
+  --port 30333 \
```

Comment:
```suggestion
  --port 30333 \
  --listen-addr "/ip4/0.0.0.0/tcp/30333/ws" \
```
This makes the collator listen on websocket, future proofing to make interaction with light clients easier